dd-sdk-android/src/test/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayRecordCallbackTest.kt

```diff
@@ -36,30 +44,12 @@ internal class SessionReplayRecordCallbackTest {
     }
 
     @Test
-    fun `M update session replay context W onStartRecording`() {
-        // When
-        testedRecordCallback.onStartRecording()
-
-        // Then
-        argumentCaptor<(MutableMap<String, Any?>) -> Unit> {
-            verify(mockDatadogCore).updateFeatureContext(
-                eq(SessionReplayFeature.SESSION_REPLAY_FEATURE_NAME),
-                capture()
-            )
-
-            val featureContext = mutableMapOf<String, Any?>()
-            lastValue.invoke(featureContext)
+    fun `M update session replay context W onRecordForViewSent`(@Forgery forge: Forge) {
```

Comment:
`Forgery` annotation is not needed here

```suggestion
    fun `M update session replay context W onRecordForViewSent`(forge: Forge) {
```